bears/stylus/StylintBear.py

```diff
@@ -36,12 +42,230 @@ class StylintBear:
+                        colors: bool=True,
+                        spaces_after_commas: bool=True,
+                        spaces_after_comments: bool=True,
+                        trailing_whitespace: bool=False,
+                        css_literal: bool=False,
+                        max_selector_depth: bool=False,
+                        check_duplicates: bool=True,
+                        efficient_properties: bool=True,
+                        extend_preference: str='False',
+                        indent_size: int=0,
+                        leading_zero: bool=False,
+                        max_errors: int=0,
+                        max_warnings: int=0,
+                        mixed_spaces_and_tabs: bool=False,
+                        variable_naming_convention: str='False',
+                        class_naming_convention: bool=False,
+                        none_keyword: bool=False,
+                        check_no_important_keyword: bool=True,
+                        spaces_inside_parentheses: bool=None,
+                        placeholder: bool=True,
+                        prefix_vars_with_dollar: bool=True,
+                        semicolons: bool=False,
+                        sort_order: str='alphabetical',
+                        stacked_properties: bool=False,
+                        check_valid_property: bool=True,
+                        zero_units: bool=False,
+                        z_index_normalize: bool=False,
+                        stylint_config: str=''):
+        """
+        :param block_keyword:
+            When ``True`` expect the ``@block`` keyword when defining block
+            variables. When ``False``, expect no ``@block`` keyword when
+            defining block variables. When ``None``, do not check either way.
+            Supported values are ``True``, ``False`` and ``None``.
+        :param brackets:
+            When ``True``, expect ``{}`` when declaring a selector. When
+            ``False``, expect no brackets when declaring a selector.
+            Supported values are ``True``, ``False`` and ``None``.
+        :param colons:
+            When ``True``, expect ``:`` when declaring a property. When
+            ``False``, expect no ``:`` when declaring a property.
+            Supported values are ``True``, ``False`` and ``None``.
+        :param colors:
+            When ``True``, enforce variables when defining hex values.
+            Supported values are ``True`` and ``None``.
+        :param spaces_after_commas:
+            Enforce or disallow spaces after commas.
+            Supported values are ``True``, ``False`` and ``None``.
+        :param spaces_after_comments:
+            Enforce or disallow spaces after line comments.
+            Supported values are ``True``, ``False`` and ``None``.
+        :param trailing_whitespace:
+            If ``None``, ignores trailing white space. If ``False``, trailing
+            white space will throw a warning.
```

Comment:
Change None, False to True, False and rename the setting to fit those values better^^